### PR TITLE
added interview of aadhitya-amarendiran

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c5d9a697a15a16e700b4c708582a674bf63d0f69 at: https://6717a5b1bef3181f9b764d6c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d81a864d3020a12fb69e973fad692448dfc34eaf at: https://6718ff9d1519c70071abba6f--layer5.netlify.app

**Comment by leecalcote** [comment]
Yay!

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    What is your favorite Meshery CLI Command?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>Oof! That’s a tricky one. But my favorite one is definitely mesheryctl perf</p>
```
Comment (alphaX86) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:115]:
```suggestion
  <p>Oof! That’s a tricky one. But my favorite one is definitely <i>mesheryctl perf</i></p>
```
I made this italicized in doc, maybe we can make the same here...

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>What is your hot tip for working with Meshery that others may not know?</p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    If you’re starting out with Meshery, make sure to use the Meshery Playground if you want to get hands-on for the first time without the need to deploy Meshery in your system. After you get the basics right, install Meshery and log in to your deployed instance. You’ll see that your designs,performance test results and configurations remain intact in your instance as if they are present exactly the same in the Playground. There’s no need to start from scratch. Just continue where you left off!
```
Comment (alphaX86) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:126]:
```suggestion
    If you’re starting out with Meshery, make sure to use the Meshery Playground if you want to get hands-on for the first time without the need to deploy Meshery in your system. After you get the basics right, install Meshery and log in to your deployed instance. You’ll see that your designs, performance test results and configurations remain intact in your instance as if they are present exactly the same in the Playground. There’s no need to start from scratch. Just continue where you left off!
```

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    Your most often used emoji? Your preference: movie or book? Morning person
+    or night owl? What have you worked on in the past six months that you’re
+    particularly proud of?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+   Most used emoji: 😎. I’m a morning person usually and sleep early, but sometimes I’m a night owl when it comes to intense work. I’d prefer movies compared to books as for some they clearly adapt from books. I’ve worked on refactoring the mesheryctl pattern command to mesheryctl design without losing the core features present. This took me a bit of time as I had to balance my current work as well which caused a bit of inactivity. But I managed to complete it, and I’m proud of doing such great work!
```
Comment (alphaX86) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:154]:
```suggestion
   Most used emoji: 😎. I’m a morning person usually and sleep early, but sometimes I’m a night owl when it comes to intense work. I’d prefer movies compared to books as for some they clearly adapt from books. I’ve worked on refactoring the <i>mesheryctl pattern</i> command to <i>mesheryctl design</i> without losing the core features present. This took me a bit of time as I had to balance my current work as well which caused a bit of inactivity. But I managed to complete it, and I’m proud of doing such great work!
```

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+  <span>Vivek:</span>
+  <p>
+    In other words, whether your contribution is big or small, it sounds like aiming for high-quality contributions that add value to the projects is key.
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    Yes, you got it right! Even the smallest contribution which creates a good impact on the project is the main key to Open source. All that matters is perseverance, challenging yourself to limits and learning. Do these things right and you’ll find yourself growing in the community.
```
Comment (alphaX86) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:183]:
```suggestion
    Yes, you got it right! Even the smallest contribution which creates a good impact on the project becomes the big factor in any Open source project. All that matters is perseverance, challenging yourself to limits and learning. Do these things right and you’ll find yourself growing in the community.
```
Making bit corrections here...

**Review by alphaX86 — CHANGES_REQUESTED** [review]
Reviewed, it's great! Made some corrections from my side. Not sure if <i> tag works here...

**Comment by vishalvivekm** [comment]
@hargunkaur286 
Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+  <span>Vivek:</span>
+  <p>
+    In other words, whether your contribution is big or small, it sounds like aiming for high-quality contributions that add value to the projects is key.
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    Yes, you got it right! Even the smallest contribution which creates a good impact on the project becomes the big factor in any to Open source project. All that matters is perseverance, challenging yourself to limits and learning. Do these things right and you’ll find yourself growing in the community.
```
Comment (alphaX86) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:183]:
```suggestion
    Yes, you got it right! Even the smallest contribution which creates a good impact in a project becomes a great factor in Open source. All that matters is perseverance, challenging yourself to limits and learning. Do these things right and you’ll find yourself growing in the community.
```

Minor correction here. Rest seems fine for now.

**Comment by l5io** [comment]
🚀 Preview for commit ef70701b9f977ad5727a128f8ff8298986c0361e at: https://6725b8fadf9db00ed2cd595e--layer5.netlify.app

**Review by alphaX86 — APPROVED** [review]
LGTM 👍 great job @hargunkaur286

File: src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    What is your favorite Meshery CLI Command?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>Oof! That’s a tricky one. But my favorite one is definitely <b>mesheryctl perf</b></p>
```
Comment (vishalvivekm) [inline src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx:115]: `mesheryctl perf` 
I wonder if it makes sense to wrap these commands in `<code></code>` instead :thinking:

**Comment by l5io** [comment]
🚀 Preview for commit 3a415cc2d2674d231dc3643ebc95ef7f9b1a8586 at: https://672744cd667d6adac26b3a27--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]
lets go

**Pull request merged**